iib/web/api_v1.py

```diff
@@ -501,7 +543,7 @@ def regenerate_bundle():
     error_callback = failed_request_callback.s(request.id)
     try:
         handle_regenerate_bundle_request.apply_async(
-            args=args, link_error=error_callback, argsrepr=repr(safe_args), queue=_get_user_queue(),
+            args=args, link_error=error_callback, argsrepr=repr(safe_args), queue=_get_user_queue()
```

Comment:
Unnecessary change?

Reply:
Python black complains if I don't do this

Reply:
Maybe a separate commit then?